### DISCUSSION TITLE
Welcome to Google-Assistant-Unofficial-Desktop-Client Discussions!

### DISCUSSION THREAD

**Discussion by Melvin-Abraham** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#191] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LavecTech** [comment]
Hi Melvin, I'm a web developer from Sydney. Great work building this. I always though it would be good to have Google assistant running on your computer as it is more than capable of doing what a smart speaker can do. How are you going with having the program listen for "ok google" as well as the keyboard shortcut?

**Reply by Melvin-Abraham** [reply]
@LavecTech I am planning to use **PicoVoice** for wake-word detection. Do you have any ideas or opinions regarding this? If yes, do share it here... 😊

**Comment by windowsvn** [comment]
Can you add font style option in settings.. (support for Unicode or default OS font). The current font does not support well in my country. Thanks.  (Sr myEng)

**Reply by Melvin-Abraham** [reply]
@windowsvn Can you share the language or the locale that you are using on the app?

**Reply by windowsvn** [reply]
Vietnamese, sir

**Reply by Melvin-Abraham** [reply]
@windowsvn Vietnamese seems to render without any unicode issues. Can you share a screenshot to convey the issue?

**Comment by sandi7777777** [comment]
I really like your work and I've been tested it on my pc, but I wonder how do I clear the data from this application, I need it because I want to setup it again from the first since I want to change my google account. 

I tried to uninstall it and then I installed it again, it still using my old data. 
Do you know how?

**Reply by Melvin-Abraham** [reply]
@sandi7777777 You only need to change the **Key File Path** as per your Google account and clear the **Saved Tokens Path** to fetch a new one. No other settings needs to be reset if don't want to.

**Comment by Jaskaran1998** [comment]
hi, is there a way to change the hotkey to launch google assistant??

**Reply by Jaskaran1998** [reply]
> hi, is there a way to change the hotkey to launch google assistant??

i m using a macro to change hotkey, but still, is there a way?

**Reply by esau-morais** [reply]
This feature was already implemented in the build version, but not available yet to the latest release that you've probably just installed. In the meantime, I recommend to [build it in your local machine](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/blob/master/CONTRIBUTING.md#how-to-build)

**Comment by windowsvn** [comment]
Today, GAssistant has been very slow to respond (deplay). 
Status is loaded until it quit the tray to start action

(Sorry I'm not good at English)

**Comment by benjaminwtwall** [comment]
Hi, I can't view pictures, is that an indev feature? Also, I assume there's no way for me to integrate this with my home devices?

**Reply by Melvin-Abraham** [reply]
@benjaminwtwall

> I can't view pictures, is that an indev feature

No, this is not specific to any indev feature. This was initially a part of the Assistant SDK and used to work just fine within the app. Unfortunately, over time Google has altered the responses/behavior of the SDK due to which this does not work anymore.

> I assume there's no way for me to integrate this with my home devices

If you mean controlling smart devices supported by Google Home, then probably yeah, you can control them (I do not own a Google Home supported smart home device to test if that works now) but if you are referring to the Speaker Group feature, this is currently not available and is planned for future.

**Comment by VasigaranAndAngel** [comment]
Hello Melvin, This project is awesome. I like to help to make this project more better. I'm just going to torture you with new ideas. 😊

**Reply by Melvin-Abraham** [reply]
@VasigaranAndAngel You are more than welcome to make contributions in the form of Feature Requests. If you directly want to contribute to this project, you may go through the [contribution guidelines](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/blob/master/CONTRIBUTING.md).
